tests/integration/test_main.py

```diff
@@ -0,0 +1,52 @@
+from fastapi.testclient import TestClient
+from unittest.mock import patch
+from main import app
+import json
+
+
+# Create a test client for the FastAPI app
+client = TestClient(app)
+
+
+def test_search_issues(s3_bucket, duckdb_connection):
+    # Prepare test params
+    params = {
+        "dataset": "conservation-area",
+        "offset": 0,
+        "limit": 10,
+    }
+
+    # Test the function that interacts with DuckDB and S3 via LocalStack
+    with patch("db.duckdb.connect", return_value=duckdb_connection):
```

Comment:
Why do we need to patch this method and intercept the duckdb_connection?  Ideally the integration test would just let this happen?